### PR TITLE
Update AUTHORS

### PR DESCRIPTION
**Pull request by shivareddyiirs**
## Description
Include a few sentences describing the overall goals for this PR (pull request). If applicable also add screenshots.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

### CODE REVIEW

**Comment by 3nids** [comment]
Authors is listing people with commit rights (and is listed by order of appearance).
So contributors is a more appropriate place for your [contribution](https://github.com/qgis/QGIS/commits?author=shivareddyiirs).

File: doc/AUTHORS [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #
 # ALWAYS USE UTF-8 WHEN WRITING THIS FILE!
 #
+Shiva Reddy Koti <kotishiva at gmail dot com>
```
Comment (elpaso) [inline doc/AUTHORS]: maybe better move to the end

Comment (shivareddyiirs) [reply doc/AUTHORS]: Sure. my mistake. changed

**Review by 3nids — CHANGES_REQUESTED** [review]
as written in the comment, authors is for people with commit rights.

**Comment by shivareddyiirs** [comment]
Thanks. I will raise another PR with only changes in contributors list

**Pull request closed without merging**